.github/workflows/build.yml

```diff
@@ -0,0 +1,80 @@
+name: Build Firmware - PlatformIO CI
+
+on:
+  repository_dispatch:
+    types: [build-firmware]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Tag
+        id: tag
+        uses: JinoArch/get-latest-tag@latest
+
+      - uses: actions/checkout@v3
+        with:
+            submodules: secplus
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/pip
+            ~/.platformio/.cache
+          key: ${{ runner.os }}-pio
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+      - name: Install PlatformIO Core
+        run: |
+            pip install --upgrade pip
+            pip install --upgrade platformio
+
+      - name: Install External Dependencies
+        run: |
+          cd lib
+          git clone https://github.com/argilo/secplus.git
```

Comment:
We shouldn't need to do this if the `submodules: secplus` works above.

Reply:
When I tried it without this, it didn't work. So, I included this.

Reply:
I think you need `submodules: true` instead of `submodule: secplus`.

Reply:
nice! I correct that with #57.